sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/implementation/PartitionProcessorImpl.java

```diff
@@ -128,16 +128,20 @@ public PartitionProcessorImpl(ChangeFeedObserver observer, ChangeFeedContextClie
                         BridgeInternal.getPartitionKeyInternal(this.options.getPartitionKey()), Thread.currentThread().getId(), clientException);
                     StatusCodeErrorType docDbError = ExceptionClassifier.classifyClientException(clientException);
 
+                    // Note: @kushagraThapar and @moderakh to ensure fallthrough is not expected and break is good
```

Comment:
I am not sure about this one. 
@milismsft  - can you please look at this `break` statement introduction change ?

Reply:
Looks good.